microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -169,8 +169,7 @@ You can delete templates stored on the device and clear the cache data. Go to `e
 
 <!-- Above, should pick one way. Check UI for privacy updates -->
 
-You can use the title of your collection to find relevant Pinterest Topic pages when searching in Microsoft Bing. Go to `edge://settings/privacy`. Turn on the **Show suggestions from Pinterest in Collections** setting. Microsoft Edge does not send data about your collections to Pinterest. You can remove the suggestions and stop searches for Pinterest Topic pages. Go to `edge://settings/privacy` and turn off the **Show suggestions from Pinterest in Collections** setting.
-Collections aren't available when using **InPrivate** or **Guest** modes.
+If you're signed in with a **personal Microsoft account**, your collections, notes, comments, and images are automatically saved on Microsoft cloud for free.  If you want to turn off this service, go to `edge://settings/privacy`, <!--todo: scroll down to which section?--> and then turn off the **Show Collections and follow content creators in Microsoft Edge**<!--todo: no such setting--> toggle.  When this setting is turned off, Collections are disabled on this device.  Collections aren't available when using Guest mode or InPrivate browsing for your **personal Microsoft account**.
```

Comment:
I recommend removing "personal" here, and then adding a section about the behavior for when your'e signed in with a work or school account. 

```suggestion
If you're signed in with a **Microsoft account**, your collections, notes, comments, and images are automatically saved on Microsoft cloud for free.  If you want to turn off this service, go to `edge://settings/privacy`, <!--todo: scroll down to which section?--> and then turn off the **Show Collections and follow content creators in Microsoft Edge**<!--todo: no such setting--> toggle.  When this setting is turned off, Collections are disabled on this device.  Collections aren't available when using Guest mode or InPrivate browsing for your **personal Microsoft account**.
```

Reply:
@yuanjunz - see the above Comment.  I don't know if you're able to push a commit to this PR that I created (maybe I should have created a forked-repo PR and then Invited you to contribute to my forked repo).  

You can paste draft text here, or to go fresh Files Changed tab of this PR and then enter a Comment that's a Suggested Change (click [+/-] icon in comment's toolbar).

The draft text could be a paragraph, about the behavior for when you're signed in with a work Microsoft account or school Microsoft account instead of personal Microsoft account.

Another article has wording:
Use a personal Microsoft account (MSA) (not a work Microsoft account (MSA) or a school Microsoft account (MSA)) to register
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/d1d0843e-e715-474c-b9c8-13bc77af232a)

Reply:
Agree to removing "personal" here, use **Microsoft account**

Reply:
removed 'personal' 2x.
Didn't do:
* Add a section about the behavior for when you're signed in with a work or school account.

Reply:
"Microsoft account" occurs 23x in https://learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/, not bold.